app-near/src/server/typeform/handleAnswer.ts

```diff
@@ -106,3 +71,132 @@ export const handleAnswer = async (req: NextRequest): Promise<NextResponse> => {
     );
   }
 };
+
+const handleSuForm = async (
+  parsedBody: TypeformWebhookPayload,
+  answers: Record<string, string | boolean>,
+  formId: string,
+): Promise<
+  NextResponse<{ message: string }> | NextResponse<{ error: string }>
+> => {
+  // should be part of neighborhood
+  if (isPartOfNeighborhood(answers)) {
```

Comment:
:red_circle: devrait plutôt être
```suggestion
  if (!isPartOfNeighborhood(answers)) {
```